src/ice_ridge.F90

```diff
@@ -239,7 +248,11 @@ subroutine ice_ridging(IST, G, IG, mca_ice, mca_snow, mca_pond, TrReg, CS, US, d
 
   call icepack_query_tracer_sizes(ncat_out=ncat_out,ntrcr_out=ntrcr_out, nilyr_out=nilyr_out, nslyr_out=nslyr_out)
 
-  if (nIlyr .ne. nilyr_out .or. nSlyr .ne. nslyr_out ) call SIS_error(FATAL,'nilyr or nslyr mismatch with Icepack')
+  if (nIlyr .ne. nilyr_out .or. nSlyr .ne. nslyr_out ) &
```

Comment:
This indentation block needs to be fixed.
(We generally require >2 space indentation for line continuations.  The one exception is after an if-block, which is only 2 spaces.   So the first line should be 2 spaces, not one, and the other lines should be "more than 2" (your discretion).